src/components/Stepper/index.js

```diff
@@ -22,80 +23,54 @@ const disabledStyles = ({ disabled, theme }) =>
   `
 
 const containerSizes = {
-  large: '48px',
-  medium: '40px',
-  small: '32px',
+  large: 48,
+  medium: 40,
+  small: 32,
 }
 
-const buttonSizes = {
-  large: '16px',
-  medium: '10px',
-  small: '4px',
+const iconSizes = {
+  large: 26,
+  medium: 24,
+  small: 22,
 }
 
 const StyledTouchable = styled(Touchable, {
   shouldForwardProp: prop => !['position', 'size'].includes(prop),
 })`
-  border: 1px solid ${({ theme }) => theme.colors.gray350};
-
   justify-content: center;
   align-items: center;
-  height: 100%;
-  padding-right: 16px;
-  padding-left: 16px;
+  height: calc(100% - 8px);
   position: relative;
+  border-radius: 4px;
+  width: ${({ size }) => containerSizes[size] - 10}px;
 
-  :hover,
-  :focus {
-    > svg {
-      fill: ${({ disabled, theme }) => !disabled && theme.colors.primary};
-    }
+  > svg {
+    fill: ${({ disabled, theme }) => !disabled && theme.colors.primary};
   }
 
-  ${({ position, size, theme }) =>
-    position === 'left'
-      ? `
-      border-right-width: 0;
-      border-top-left-radius: 4px;
-      border-bottom-left-radius: 4px;
-      border-radius: 4px 0 0 4px;
-      padding: 0 ${buttonSizes[size]};
-      border-right: ${size === 'large' ? '0' : '1px'} solid ${
-          theme.colors.gray350
-        };
-    `
-      : `
-          border-left-width: 0;
-          border-radius: 0 4px 4px 0;
-          padding: 0 ${buttonSizes[size]};
-          border-left: ${size === 'large' ? '0' : '1px'} solid ${
-          theme.colors.gray350
-        };
-    `}
-`
+  :hover:not([disabled]) {
+    background: ${({ theme }) => theme.colors.gray200};
+  }
 
-const StyledSeparator = styled('span', {
-  shouldForwardProp: prop => !['size'].includes(prop),
-})`
-  position: absolute;
-  font-size: 30px;
-  top: 7px;
-  font-weight: 500;
-  color: ${({ theme }) => theme.colors.gray350};
-  ${({ size }) => (size !== 'large' ? 'display: none' : '')};
+  margin: 0 4px;
 `
 
 const StyledCenterTouchable = styled(Touchable)`
   flex: 1;
   flex-direction: row;
-  height: 100%;
+  height: calc(100% - 8px);
   align-items: center;
   outline: none;
   justify-content: center;
-  border: 1px solid ${({ theme }) => theme.colors.gray350};
-
-  border-left-width: 0;
-  border-right-width: 0;
+  border-radius: 4px;
+  :hover:not([disabled]):not(:focus) {
```

Comment:
:wat: 😂 

There is no easier selector ?

Reply:
I can change it for `:hover:not([disabled], :focus)` if you prefer 😄

Reply:
That's just slightly better but yeah 😄